packages/rn-tester/RNTester/AppDelegate.mm

```diff
@@ -139,10 +139,11 @@ - (void)userNotificationCenter:(UNUserNotificationCenter *)center
 
 #pragma mark - New Arch Enabled settings
 
-- (BOOL)bridgelessEnabled
-{
-  return [super bridgelessEnabled];
-}
+// Should we support this?
+//- (BOOL)bridgelessEnabled
+//{
+//  return [super bridgelessEnabled];
+//}
```

Comment:
yes, this is used internally to swizzle the value in some tests

Reply:
Okay, Im going to support this then!